FluentUI/build.gradle

```diff
@@ -73,7 +74,7 @@ publishing {
                 configurations.implementation.allDependencies.each {
                     if (it.group != null && (it.name != null && it.name != "unspecified") && it.version != null) {
                         def dependencyNode = dependenciesNode.appendNode('dependency')
-                        if (it.group.equals('fluentui-android')) {
+                        if (it instanceof ProjectDependency) {
```

Comment:
This was missing in the first PR, but we added exactly same change and merged it resolve the issue there. This was done in PR #51 . I added it here too as it will be required for CCB as well